index.html

```diff
@@ -492,12 +540,19 @@ <h4 id="return-values">Return Value</h4>
 </pre>
     </section>
     <section id="privacy" class='informative'>
-      <h2>Privacy</h2>
-      <p>This specification does not add extra security or privacy
-      considerations in addition to the ones associated with forms submissions
-      in HTML, as noted <a class="external" href=
-      "http://www.w3.org/TR/html5/introduction.html#fingerprint" title=
-      "HTML privacy concerns">here</a>.</p>
+      <h2>Privacy and Security</h2>
+      <p>The <code>sendBeacon</code> interface provides an asynchronous and
+      non-blocking mechanism for delivery of data. It is designed to eliminate
+      the need for the widespread use of expensive anti-patterns that block
+      user interaction, and to enable more efficient scheduling and use of
+      system resources. The same functionality is possible today, albeit with
+      high performance costs and degraged user experience.</p>
+      <p><code>sendBeacon</code> does not add new privacy considerations. However, implementers should take the following precautions:</p>
```

Comment:
The "delayed sending" scenario, which can reveal browsing habits over a previous network is something we should probably add here.